src/Sulu/Bundle/ContactBundle/Entity/SocialMediaProfile.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use JMS\Serializer\Annotation\Exclude;
+use JMS\Serializer\Annotation\Groups;
+use Sulu\Component\Contact\Model\ContactInterface;
+
+/**
+ * SocialMediaProfile.
+ */
+class SocialMediaProfile
```

Comment:
if you'd add `@ExclusionPolicy("All")` you wouldn't have to do the excludes below